docs/using-the-jdbc-driver/using-plugins/UsingTheLimitlessConnectionPlugin.md

```diff
@@ -0,0 +1,42 @@
+# Using the Limitless Connection Plugin
+
+## What is Amazon Aurora Limitless Database?
+
+Amazon Aurora Limitless Database allows users to horizontally scale their Aurora Databases, handling millions of write transactions per second and managing petabytes of data. To learn more, please [read Amazon Aurora Limitless documentation](// TODO).
+
```

Comment:
I think you could add here something along the lines of

"Users will be able to use the AWS JDBC Driver with Aurora Limitless just like any other Aurora database. However some configurations need to be set to make things work properly/as expected"